sunpy/net/base_client.py

```diff
@@ -128,6 +128,10 @@ def _print_client(client, html=False):
     t = Table(names=["Attr Type", "Name", "Description"],
               dtype=["U80", "U80", "U80"])
     for client_key in attrs.keys():
+        # Work around for * attrs having one length.
+        if len(attrs[client_key]) == 1 and attrs[client_key][0] == "*":
+            t.add_row((client_key.__name__, "All", "All valid attributes"))
+            continue
```

Comment:
A large hack.

Reply:
If somehow we got a '*' and a valid value, this hack would break.

Reply:
Why not move this into the loop below?
```python
for name, desc in attrs[client_key]:
    if name == "*":
        t.add_row((client_key.__name__, "All", "All valid attributes"))
        continue
    t.add_row((client_key.__name__, name, desc))
```
?

Reply:
Do it.

Reply:
The issue with this was that if the length is one, there is no "desc" to loop over and thus errored.